Makefile

```diff
@@ -69,7 +69,8 @@ NEW_URLS=$(shell scripts/ignore_new_urls.sh 2> /dev/null)
 URL_IGNORE:=$(URL_IGNORE)$(NEW_URLS)
 
 ## validate-site: Builds the site and validates the pages. This is used for CI
+CONFIG='{\"enforce_https\": false,\"check_internal_hash\": false, \"check_external_hash\":false,\"ignore_empty_alt\":true}'
```

Comment:
Where is this CONFIG used? These options look like the same as what is being passed in as cmd line args on line 75. I think you should remove this line 72 that defines CONFIG since it looks like you are passing these values in directly on the command line (and additional ones) on line 75.
```suggestion
```

Reply:
Yes, that's correct. I was using this to test if we can workaround the bug in 4.2 somehow, but I forgot to remove it later...